automation/taskcluster/lib/tasks.py

```diff
@@ -86,6 +97,29 @@ def craft_compare_locales_task(self):
             command='pip install "compare-locales>=5.0.2,<6.0" && compare-locales --validate l10n.toml .'
         )
 
+    def craft_sign_task(self, build_task_id, artifacts, component_name, is_staging):
+        payload = {
+            "maxRunTime": 600,
+            "upstreamArtifacts": [{
+                "paths": [artifact["taskcluster_path"] for artifact in artifacts],
+                "formats": ["autograph_gpg"],
+                "taskId": build_task_id,
+                "taskType": "build"
+            }]
+        }
+
+        return self._craft_default_task_definition(
+            worker_type='mobile-signing-dep-v1' if is_staging else 'mobile-signing-v1',
+            provisioner_id='scriptworker-prov-v1',
+            dependencies=[build_task_id],
+            routes=[],
+            scopes=[
+                "project:mobile:android-components:releng:signing:cert:{}-signing".format("dep" if is_staging else "release"),
```

Comment:
@JohanLorenzo You were right, I double-checked too in https://github.com/mozilla-releng/build-puppet/blob/master/modules/signing_scriptworker/templates/passwords.json.erb#L23 vs https://github.com/mozilla-releng/build-puppet/blob/master/modules/signing_scriptworker/templates/passwords.json.erb#L47 and they are the same. Rather than adding a new layer of confusion, I'd say we keep `release-signing` for snapshots as well. What do you think?

Reply:
Fine by me!